app/scripts/controllers/modals/editModal.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+/* jshint unused: false */
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:EditModalController
+ * @description
+ * # ProjectController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+    .controller('EditModalController', function($scope, APIService, DataService, QuotaService, $uibModalInstance, $filter) {
+
+        var amountAndUnit = $filter('amountAndUnit');
+        var usageWithUnits = $filter('usageWithUnits');
+        var usageValue = $filter('usageValue');
+
+        var allocatedAmountAndUnit = amountAndUnit($scope.currentCapacity);
+
+        $scope.claim = {}
+        $scope.claim.capacity = parseInt(allocatedAmountAndUnit[0]);
+        $scope.claim.unit = allocatedAmountAndUnit[1];
+
+        $scope.currentCapacityUnits = angular.copy($scope.claim);
+
+        $scope.claim.capacity += 1;
+
+        $scope.units = [{
```

Comment:
OK. Even so, do we know that it will always come back from the API with that unit? If I request 1.25GiB, can the API server return the values in MiB to avoid the decimal, and what happens in this editor?

(I have no idea which is why I'm asking :)